script.js

```diff
@@ -208,6 +208,11 @@ $(".btn").on("click", function (event) {
         var chart = new google.visualization.LineChart(document.getElementById('curve_chart'));
 
         chart.draw(data, options);
```

Comment:
Andrew is the chart being responsive? Here's what I thought it was supposed to be with regard to the closing braces and parentheses:
chart.draw(data, options);
      }
      $(window).resize(function(){
        drawChart();
      });
    });

Reply:
I adjusted the code so that is looks like the code in your comment. It seemed to be working fine, but it's also working fine with the code above.

Reply:
got it, looking good!